.github/workflows/release.yml

```diff
@@ -6,32 +6,35 @@ on:
       - beta
 jobs:
   release:
+    permissions:
```

Comment:
You'll need the `id-token` for NPM provenance, which I'd like to make a standard practice across XMTP

Reply:
good catch! i added it back.